l10n/en/firefox/browsers/compare/chrome.ftl

```diff
@@ -13,10 +13,16 @@ compare-chrome-we-compare-mozilla-firefox = We compare { -brand-name-mozilla } {
 # Main title.
 compare-chrome-comparing-firefox-browser = Comparing { -brand-name-firefox-browser } with { -brand-name-google } { -brand-name-chrome }
 
+# Variables:
+#   $attrs (string) - link to https://monitor.firefox.com with additional attributes for analytics
+compare-chrome-at-firefox-we-have-a-huge-v2 = At { -brand-name-firefox }, we have a huge number of dedicated users who appreciate our steadfast dedication to online privacy. For example, the latest version of { -brand-name-firefox } includes a feature called Enhanced Tracking Protection (ETP) which is turned on by default for all users worldwide. ETP blocks over 2,000 trackers, including social trackers from companies like { -brand-name-facebook }, { -brand-name-twitter }, and LinkedIn. It also has an integrated feature called <a { $attrs }>{ -brand-name-mozilla-monitor }</a> that automatically notifies you if your password has been breached or needs to be updated. In addition to these protections, Private Browsing mode in { -brand-name-firefox } automatically deletes your browsing information such as history and cookies, leaving no trace after you finish your session.
```

Comment:
Let's take this opportunity to stop using the `-brand-name-facebook` and `-brand-name-twitter` keywords in the v2 string.